videojs-contrib-dash.html

```diff
@@ -17,19 +17,14 @@
   <script src="js/docs-themes.js" type="text/javascript"></script>
   <script type="text/javascript">set_style();</script>
 
-  <!-- Chromecast pre-load -->
-  <head data-cast-api-enabled="true"> <!-- allow chromecast extention -->
-  <script src="//www.gstatic.com/cv/js/sender/v1/cast_sender.js"></script>
-
-  <!-- VideoJS, DashJS -->
-  <script src="//vjs.zencdn.net/5.10/video.js"></script>
-  <script src="//cdn.dashjs.org/latest/dash.all.debug.js"></script>
-  <script src="https://npmcdn.com/videojs-contrib-dash@^2.6.1/dist/videojs-dash.js"></script>
-  <link href="//vjs.zencdn.net/5.10/video-js.css" rel="stylesheet">
+  <!-- Video.js, DashJS -->
+  <script src="//vjs.zencdn.net/7.16.0/video.js"></script>
+  <script src="https://unpkg.com/videojs-contrib-dash@5.1.0/dist/videojs-dash.js"></script>
+  <link href="//vjs.zencdn.net/7.16.0/video-js.css" rel="stylesheet">
 
   <!-- Chromecast post-load -->
-  <script src="js/videojs-chromecast/videojs-chromecast.min.js"></script>
```

Comment:
Can you `rm` these files from the code base too as it's being replaced with the CDN version?